src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageSearch/JsonFormat/NormalVerbosityJsonFormatSearchResultPackage.cs

```diff
@@ -6,17 +6,17 @@
 
 namespace NuGet.CommandLine.XPlat
 {
-    internal class NormalVerbosityPackage : MinimalVerbosityPackage
+    internal class NormalVerbosityJsonFormatSearchResultPackage : MinimalVerbosityJsonFormatSearchResultPackage
     {
-        [JsonPropertyName("total downloads")]
+        [JsonPropertyName("totalDownloads")]
         public long? Downloads { get; set; }
 
         [JsonPropertyName("owners")]
         public string Owners { get; set; }
```

Comment:
I recently merged a change to Protocol that you could consider echoing here. The property is `OwnersList` and provides an enumerable deserialized from the JSON. The `Owners` is representing a comma-delimited list of the items in the `OwnersList`.
https://github.com/NuGet/NuGet.Client/blob/001ea1a/src/NuGet.Core/NuGet.Protocol/Model/PackageSearchMetadata.cs#L67

Reply:
Yes, we could have an `ownersList` property instead of `owners`. However, should we do that in a separate PR?